SECURITY.md

```
@@ -0,0 +1,53 @@
+For the complete Ansible Security Policy, including what qualifies as a
+reportable vulnerability, severity classification, the response process,
+coordinated disclosure, and the vulnerability management process, see
+[ansible.com/security](https://ansible.com/security).
+
+## EU Cyber Resilience Act: Open-Source Steward Statement
+
+This project is stewarded by **Red Hat, Inc.**, an open-source software steward
+as defined in Article 3(14) of the
+[EU Cyber Resilience Act (Regulation 2024/2847)](https://eur-lex.europa.eu/eli/reg/2024/2847/oj/eng).
+
+Contact: [cra-steward@redhat.com](mailto:cra-steward@redhat.com)
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)